app/styles/app.scss

```diff
@@ -2,6 +2,7 @@
 $c-black: #323131;
 $c-white: #fff;
 $c-blue: #ad5235;
+$c-blue-link: #f06000;
```

Comment:
I think this is not actually blue, right? It's orange. I guess this, and similar colors in this file, are artifacts of migrating from statechannels.org.

We should make an issue to go back and rename variables like this, to avoid confusion in future.

Reply:
#28